### PR TITLE
KAH-22 game-settings

### PR DESCRIPTION
**Pull request by WojciechKaluzinski**

### CODE REVIEW

File: lib/features/settings/views/settings_page.dart [hunk 1]
```diff
@@ -1,12 +1,101 @@
+import 'package:flutter/cupertino.dart';
```
Comment (joachimschmidt) [inline lib/features/settings/views/settings_page.dart:1]: Do we need this?

File: lib/features/settings/views/settings_page.dart [hunk 2]
```diff
@@ -1,12 +1,101 @@
...
+                    textCapitalization: TextCapitalization.characters,
+                    controller: playerNameInputController,
+                    decoration: InputDecoration(
+                      border: UnderlineInputBorder(),
+                      labelText: 'Enter your username',
+                    ),
+                  ),
+                ),
+                SizedBox(height: 35),
+                Row(
```
Comment (joachimschmidt) [inline lib/features/settings/views/settings_page.dart:67]: This entire Row can be extracted to separate widget. You are using it twice so it wil make code less complicated

File: lib/features/settings/views/settings_page.dart [hunk 3]
```diff
@@ -1,12 +1,101 @@
...
-          child: Text("Settingsy"),
+        child: BlocBuilder<SettingsCubit, Settings>(
+          builder: (context, settings) {
+            return Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                SizedBox(height: 35),
+                Center(
+                  child: Text(
+                    settings.playerName == null ? "PLAYERNAME" : settings.playerName.toString(),
```
Comment (joachimschmidt) [inline lib/features/settings/views/settings_page.dart:50]: isn't settings.playerName already String?

**Review by joachimschmidt — CHANGES_REQUESTED** [review]
When I enter SettingsPage and i previously provided my username textField starts empty, and when i click on it the username in Settings  logic becomes empty to :( I want this textField to start with value(my username) if it's possible(if username!=null)

File: lib/features/settings/views/settings_page.dart (hunk 1 above)
Comment (WojciechKaluzinski) [reply lib/features/settings/views/settings_page.dart:1]: Ready

File: lib/features/settings/views/settings_page.dart (hunk 3 above)
Comment (WojciechKaluzinski) [reply lib/features/settings/views/settings_page.dart:50]: Ready

File: lib/features/settings/views/settings_page.dart (hunk 2 above)
Comment (WojciechKaluzinski) [reply lib/features/settings/views/settings_page.dart:67]: Ready

**Pull request merged**